nailgun/entity_fields.py

```diff
@@ -265,4 +266,4 @@ class URLField(StringField):
 
     def gen_value(self):
         """Return a value suitable for a :class:`URLField`."""
-        return gen_url()
+        return gen_url(subdomain=gen_alpha())
```

Comment:
@omaciel from my observation `gen_url` method give us not so many variations of inputs and such data become similar pretty fast. Let's say, it generate:
`test.com`, `example.com`, `test.biz`, `example.biz` and few more, so even for 10 tests there is chance that we will have duplicated entry. Using proposed approach we decrease such chances in hundreds times